app/src/common/shared/org/mozilla/vrbrowser/browser/engine/SessionStack.java

```diff
@@ -962,6 +963,8 @@ public void onCanGoForward(@NonNull GeckoSession aSession, boolean aCanGoForward
 
         Log.d(LOGTAG, "onLoadRequest: " + uri);
 
+        ((VRBrowserApplication) mContext.getApplicationContext()).getServices().interceptFxaUrl(uri);
```

Comment:
I left this for future reafactor and I never did it. I've change it to use NavigationDelegate LoadRequest callback to avoid a hardcoded explicit method call.